.github/workflows/ci_accountability.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   main:
     name: Tests
     runs-on: ubuntu-latest
-    if: "!startsWith(github.head_ref, 'l10n_')"
+    if: "!startsWith(github.head_ref, 'chore/l10n')"
```

Comment:
The name of the branches is set by Crowdin and I'm not sure if this can be customized!

Reply:
Yes, it's possible to customize the "service branch": https://support.crowdin.com/github-integration/ 
It's already done on the last PRs: https://github.com/decidim/decidim/pull/6226